local-tests/setup/tinny-config.ts

```diff
@@ -0,0 +1,145 @@
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+
+export enum LIT_TESTNET {
+  LOCALCHAIN = 'localchain',
+  MANZANO = 'manzano',
+  CAYENNE = 'cayenne',
+}
+
+export interface ProcessEnvs {
+  /**
+   * Each test is executed in a loop with a maximum number of attempts specified by `devEnv.processEnvs.MAX_ATTEMPTS`.
+   */
+  MAX_ATTEMPTS: number;
+
+  /**
+   * The network to use for testing. This can be one of the following:
+   * - `LIT_TESTNET.LOCALCHAIN`
+   * - `LIT_TESTNET.MANZANO`
+   * - `LIT_TESTNET.CAYENNE`
+   */
+  NETWORK: LIT_TESTNET;
+
+  /**
+   * The number of milliseconds to wait between each request.
+   */
+  DEBUG: boolean;
+
+  /**
+   * Capacity Credits: In order to execute a transaction with Lit, you’ll need to reserve capacity on the network using Capacity Credits. These allow holders to reserve a set number of requests over a desired period of time (by default expiration set to 2 days)
+   */
+  REQUEST_PER_KILOSECOND: number;
+
+  /**
+   * Wait time in milliseconds if no private keys are available.
+   */
+  WAIT_FOR_KEY_INTERVAL: number;
+
+  /**
+   * Time to wait before releasing the key after requesting it.
+   */
+  TIME_TO_RELEASE_KEY: number;
```

Comment:
when we run tests in parallel we need to use different private key to avoid nonce reuse issue, and when there are more tests than all private keys available, we need to wait for the private key being released to resue again in other tests.

Reply:
Why would we wait some time before releasing a key? Shouldn't we release it explicitly immediately when we're done using it? The JSDoc comment could be more descriptive about the 'why'.